install/openshift-logging/install.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: openshift-logging-apb
+objects:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    namespace: ${NAMESPACE}
+    name: openshift-logging-apb-admin-config
+  type: Opaque
+  data:
+    admin.kubeconfig: "${ADMIN_KUBECONFIG}"
```

Comment:
Instead of doing this, can you write RBAC rules and rolebindings for your installer?  I suspect that you don't need to do things like managing oauth access tokens and deleting nodes, which makes cluster-admin excessive for the job.

Reply:
> Instead of doing this, can you write RBAC rules and rolebindings for your installer? I suspect that you don't need to do things like managing oauth access tokens and deleting nodes, which makes cluster-admin excessive for the job.

Doing so would also allow you to use the in-cluster-config and allow proper audit trails in general

Reply:
@jcantrill @deads2k the job is kicking off a pod and using the openshift-ansible roles to install logging.

I think those roles would need changes to eliminate the need for the admin kubeconfig?

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging/library/openshift_logging_facts.py#L342

Reply:
> @jcantrill @deads2k the job is kicking off a pod and using the openshift-ansible roles to install logging.
> 
> I think those roles would need changes to eliminate the need for the admin kubeconfig?

Or you find a way to put different content there.  Something designed to run inside of a pod should use the in-cluster-config unless explicitly told not to.